### PR TITLE
Depthwise Conv, BatchNorm, Repeat & Stack Layer support

### PR DESCRIPTION
**Pull request by Ram81**
@utsavgarg @kdexd @interritus1996 This PR adds support for Depthwise Conv, Repeat & Stack layers with end to end tests. Please review.

### CODE REVIEW

File: tensorflow_app/views/import_graphdef.py [hunk 1]
```diff
@@ -186,7 +192,10 @@ def import_graph_def(request):
                 layer['params']['dim'] = str(input_dim)[1:-1]
 
             elif layer['type'][0] == 'Convolution':
-                if str(node.name) == name + '/weights' or str(node.name) == name + '/kernel':
+                if str(node.name) in [name + '/weights', 'Repeat/' + name + '/weights',
+                                      name + '/kernel', 'Repeat/' + name + '/kernel',
+                                      name + '/weights', 'Stack/' + name + '/weights',
+                                      name + '/kernel', 'Stack/' + name + '/kernel']:
```
Comment (interritus1996) [inline tensorflow_app/views/import_graphdef.py]: Aren't `name+'/weights'` and `name+'/kernel'` written twice?

**Comment by interritus1996** [comment]
If I try to export the model to Tensorflow provided in [FCStack.pbtxt](https://github.com/Ram81/IDE/blob/Layer-Support-1/example/tensorflow/FCStack.pbtxt), it throws the following error.
```
Internal Server Error: /keras/export
django     | Traceback (most recent call last):
django     |   File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/exception.py", line 39, in inner
django     |     response = get_response(request)
django     |   File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
django     |     response = self._get_response(request)
django     |   File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 187, in _get_response
django     |     response = self.process_exception_by_middleware(e, request)
django     |   File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 185, in _get_response
django     |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
django     |   File "/usr/local/lib/python2.7/dist-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
django     |     return view_func(*args, **kwargs)
django     |   File "/code/keras_app/views/export_json.py", line 185, in export_json
django     |     net[layerId], layer_in, layerId))
django     |   File "/code/keras_app/views/layers_export.py", line 129, in flatten
django     |     out[layerId] = out[layerId](*layer_in)
django     |   File "/usr/local/lib/python2.7/dist-packages/keras/engine/topology.py", line 558, in __call__
django     |     self.assert_input_compatibility(inputs)
django     |   File "/usr/local/lib/python2.7/dist-packages/keras/engine/topology.py", line 473, in assert_input_compatibility
django     |     str(K.ndim(x)))
django     | ValueError: Input 0 is incompatible with layer flatten_12: expected min_ndim=3, found ndim=2
```

File: tensorflow_app/views/import_graphdef.py (hunk 1 above)
Comment (Ram81) [reply tensorflow_app/views/import_graphdef.py]: My Bad, I'll remove it.

**Comment by Ram81** [comment]
@interritus1996 please let me check it once.

File: tensorflow_app/views/import_graphdef.py [hunk 2]
```diff
@@ -186,7 +193,9 @@ def import_graph_def(request):
                 layer['params']['dim'] = str(input_dim)[1:-1]
 
             elif layer['type'][0] == 'Convolution':
-                if str(node.name) == name + '/weights' or str(node.name) == name + '/kernel':
+                if str(node.name) in [name + '/weights', name + '/kernel',
```
[the remark this answers is not in the document]
Comment (Ram81) [reply tensorflow_app/views/import_graphdef.py]: Yeah Sure

**Comment by Ram81** [comment]
@interritus1996 @kapilkd13 @utsavgarg while merging please do #355 first.

**Pull request merged**